metadata-ingestion/src/datahub/ingestion/source/data_lake/__init__.py

```diff
@@ -246,7 +247,10 @@ def create(cls, config_dict, ctx):
 
         return cls(config, ctx)
 
-    def read_file_spark(self, file: str) -> Optional[DataFrame]:
+    def read_file_spark(self, file: str, is_aws: bool) -> Optional[DataFrame]:
+
+        if is_aws:
+            file = f"s3a://{file}"
```

Comment:
nit: s3://{file}

Reply:
or is the "a" intentional?

Reply:
Not a typo—Spark uses the s3a protocol rather than s3 (see https://spark.apache.org/docs/latest/cloud-integration.html)